src/openforms/scss/components/_form-choices.scss

```diff
@@ -11,6 +11,7 @@
     }
 
     @include element('choice') {
+      @include margin($margin: true, $value-mobile: $typography-margin-list);
```

Comment:
Added as otherwise the last child didn't have a margin.

Reply:
yeah we'll have to refactor this CSS and markup too at some point :thinking: